packages/manager/modules/pci/src/projects/project/translations/Messages_fi_FI.json

```diff
@@ -15,5 +15,6 @@
   "pci_projects_project_welcome": "Welcome to the new Public Cloud interface",
   "pci_projects_project_keystone_upgrade": "An important change will take place on 23rd June 2020 — please prepare for the upgrade to <strong>Keystone v3</strong>.",
   "pci_projects_project_keystone_upgrade_more": "Find out more",
-  "pci_projects_project_quota_warning_message": "As a security measure, your project is subject to a quota. If you would like to increase this quota, go to the <a data-ui-sref=\"{{ quotaUrl }}\">Quota and location</a> page."
+  "pci_projects_project_quota_warning_message": "As a security measure, your project is subject to a quota. If you would like to increase this quota, go to the <a data-ui-sref=\"{{ quotaUrl }}\">Quota and location</a> page.",
+  "pci_projects_maintenance_warning_message": "In order to improve stability of Public Cloud APIs, multiple maintenance will take place in the different Public Cloud regions. Operations will last for a maxim    um of 2 hours in non-working hours. During the maintenance period, it will not be possible in the concerned region to create, modify or delete Cloud resources. <br />You can find details zone by zone on this travaux task : <a href=\"http://travaux.ovh.net/?do=details&id=47661\" target=\"_blank\">http://travaux.ovh.net/?do=details&id=47661</a>"
```

Comment:
```suggestion
  "pci_projects_maintenance_warning_message": "In order to improve stability of Public Cloud APIs, multiple maintenance will take place in the different Public Cloud regions. Operations will last for a maximum of 2 hours in non-working hours. During the maintenance period, it will not be possible in the concerned region to create, modify or delete Cloud resources. <br />You can find details zone by zone on this travaux task : <a href=\"http://travaux.ovh.net/?do=details&id=47661\" target=\"_blank\">http://travaux.ovh.net/?do=details&id=47661</a>"
```